ui/component/common/form-components/form-field.jsx

```diff
@@ -4,52 +4,59 @@ import { FF_MAX_CHARS_DEFAULT } from 'constants/form-field';
 import { openEditorMenu, stopContextMenu } from 'util/context-menu';
 import * as ICONS from 'constants/icons';
 import Button from 'component/button';
+import EmoteSelector from './emote-selector';
 import MarkdownPreview from 'component/common/markdown-preview';
 import React from 'react';
 import ReactDOMServer from 'react-dom/server';
 import SimpleMDE from 'react-simplemde-editor';
+import TextareaWithSuggestions from 'component/textareaWithSuggestions';
```

Comment:
Underscore, for example ":angry_" , dismisses the suggestion popup.  It should still be there since `:angry_1` is a selection.